common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractSubscriberAndProducer.java

```diff
@@ -389,16 +389,11 @@ void drainRegular() {
                 if (t != null) {
                     try {
                         a.onError(prepareError(t));
-                    } catch (Throwable ignore) {
-                        cancel();
-                    }
+                    } catch (Throwable ignore) { }
                 } else {
                     try {
                         a.onCompleted();
-                    } catch (Throwable throwable) {
-                        cancel();
-                        a.onError(prepareError(throwable));
```

Comment:
Why was this guard deleted?

Reply:
the same -> https://github.com/grpc/grpc-java/blob/master/stub/src/main/java/io/grpc/stub/StreamObserver.java#L75